examples/xmllib/README.md

```diff
@@ -0,0 +1,39 @@
+# Guide To Try Out the New Concept of XML Helper Functions
+
+## Installation
+
+**Install Demo Version Of dsp-tools**
+
+```
+pip3 uninstall dsp-tools
+```
+
+```
+pip3 install git+https://github.com/dasch-swiss/dsp-tools.git@wip/rdu-14-replace-linebreak-with-tag
+```
+
+
+## Ontology Specifics
+
+**Classes:**
+- `:Dog`
+
+**Properties:**
+- `:hasName` -> SimpleText (1-n)
+- `:hasStreetAdress` -> RichText (1)
+
+Optional:
+- Image or IIIF-URI
+
+
+## Finishing Up
+
+**Re-Install dsp-tools**
+
+```
+pip3 uninstall dsp-tools
+```
+
+```
+pip3 install dsp-tools
```

Comment:
```suggestion
pip3 uninstall dsp-tools
pip3 install dsp-tools
```

Reply:
sorry to be controversial... but is it really too much to ask, to assume that people always use virtual environments, so that uninstalling is not needed (and you can use `python` and `pip` instead of `python3` and `pip3`?  
If not, isn't that something we should address with people, rather than cluttering our code examples?

Reply:
Unfortunately, RDU is (partially) not aware of virtualenvs, and it would first require us some effort to even convince them why they should use them. And then, we would have to explain them how to. 

But actually, teaching venvs to them is on my wishlist: https://www.notion.so/dasch-swiss/RDU-Technical-Presentations-57adafcbd46049b8b8bf516f096d725f

Reply:
I would currently leave it as it is. I did not intend for this readme to be a final version. I just copy pasted what I wrote.
Before we set up these examples, I think it is a good idea to discuss this. But I have two remarks:
- Installation should not be necessary for production examples as they should work with the latest published version.
- .venv: many prefer to have dsp-tools globally, which I understand when you think about how often they have to update it. It is unlikely that they have many other packages, so it would be a lot of maintenance work for them to use it. Additionally they may prefer to use the terminal app, and if they do we would also have to teach them how to activate a venv. That will be another hurdle. I understand why it is not ideal, but I think it is for most the best pragmatic solution.

Reply:
I can remove that here.

Reply:
it doesn't hurt either... as long as we don't lose too much time on discussing setup questions, I'm happy :)